bubblewrap.c

```diff
@@ -1473,6 +1492,13 @@ main (int    argc,
 
       /* Initial launched process, wait for exec:ed command to exit */
 
+      if (opt_block_fd != -1)
+        {
+          char b[1];
+          read (opt_block_fd, b, 1);
+          close (opt_block_fd);
+        }
+
       /* We don't need any caps in the launcher, drop them immediately. */
```

Comment:
Can we do the blocking after the parent has dropped setuid caps? It seems unsafe to have the launcher being able to start a setuid thing and keep it alive in high privileges under the launchers control.